config/webpack-config.js

```diff
@@ -27,13 +27,11 @@ module.exports = {
   plugins: [
     new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME ),
     new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME,
-                                             [ COMMON_BUNDLE_NAME ] ),
+                                             'common.[chunkhash].js' ),
     // Change warnings flag to true to view linter-style warnings at runtime.
     new webpack.optimize.UglifyJsPlugin( {
       compress: { warnings: false }
     } ),
-    // Wrap JS in raw Jinja tags so included JS won't get parsed by Jinja.
-    new BannerFooterPlugin( '{% raw %}', '{% endraw %}', { raw: true } )
```

Comment:
Woo! Looks like this can be removed from package.json: https://github.com/cfpb/cfgov-refresh/blob/flapjack/package.json#L11